docs/plugins/npe2_getting_started.md

```diff
@@ -0,0 +1,306 @@
+(npe2-getting-started)=
+
+# npe2 getting started guide
+
+```{admonition} Introducing npe2
+:class: tip
+We introduced a new plugin engine in December 2021. The new library [`npe2`][npe2]
+is a re-imagining of how napari interacts with plugins. Plugins targeting
+`napari-plugin-engine` will continue to work, but we recommend migrating to
+`npe2` as soon as possible.
+```
+
+This guide will walk you through the steps required to write a napari plugin
+from scratch.
```

Comment:
Maybe add something like "if you want to convert and existing plugin please see other document (link)"

Reply:
added to the top level admonition on that page.